docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '2'
+services:
+  game:
+    build: .
+    #image: zopanix/factorio:v0.13.13-experimental
+    volumes:
+      - ./saves:/opt/factorio/saves
+      - ./mods:/opt/factorio/mods
+    ports:
+      - "34197:34197/udp"
+    environment:
```

Comment:
Defaults for the environment are already provided in the Dockerfile, is there a need to redefine them?